src/python/ensembl/io/genomio/database/factory.py

```diff
@@ -136,6 +146,10 @@ def main() -> None:
     args = parser.parse_args()
     init_logging_with_args(args)
 
+    db_list_file = None
+    if args.db_list:
+        db_list_file = _load_multine_file(args.db_list)
```

Comment:
I may be wrong, but if I recall correctly `args.db_list` is always defined, so you could also do:
```suggestion
    if args.db_list:
        args.db_list = _load_multine_file(args.db_list)
```
And use `args.db_list` below.

Reply:
Actually `args.db_list` can be empty (in which case the factory will use the filter parameters only)